optimade/adapters/structures/ase.py

```diff
@@ -87,7 +87,9 @@ def get_ase_atoms(optimade_structure: OptimadeStructure) -> Atoms:
         if current_species.mass:
             mass = current_species.mass[0]
```

Comment:
Could probably do with an additional check for partial/multiple occupancy here, e.g.,

```suggestion
+ if len(current_species.chemical_symbols) != 1:
+     raise RuntimeError
```

Reply:
In principle, the disorder flag should be set in the `structure_features` when more than 1 element is present in a `species.chemical_symbols`.
And we already check for this. Do you want me to make an additional check for the fields that could potentially have more than 1 value ? i.e. mass and chemical symbols ?